src/controllers/notification.controller.js

```diff
@@ -0,0 +1,40 @@
+import ResponseService from '../services/response.service';
+import PreferenceService from '../services/preference.service';
+import NotificationService from '../services/notification.service';
+
+/**
+ * @class NotificationController
+ * @description Handles notifications.
+ * */
+class NotificationController {
+  /**
+   * @param  {object} req - Request object
+   *  @param {object} res - Response object
+   *  @returns {object} response
+   * */
+  static async setNotificationMode(req, res) {
+    if (req.option === 'inAppNotification') { // If request is inAppNotification
+      await PreferenceService
+        .updatePreference({ userId: req.userData.id }, { inAppNotification: req.value });
+      ResponseService.setSuccess(200, `Successfully updated your ${req.option} preferences.`);
+      return ResponseService.send(res);
+    }
+    // If request is emailNotification
+    await PreferenceService
+      .updatePreference({ userId: req.userData.id }, { emailNotification: req.value });
+    ResponseService.setSuccess(200, `Successfully updated your ${req.option} preferences.`);
```

Comment:
`req.option` does not have a user-friendly name. Use a user-friendly name instead